### PR TITLE
Added CORS to config

### PR DESCRIPTION
**Pull request by nagarajmanjunath**
* Added the CORS to grpc gateway.
* Added test case for grpc cors enable

### CODE REVIEW

File: www/grpc/server_test.go [hunk 1]
```diff
@@ -19,7 +19,6 @@ var tMockState *state.MockState
 var tMockSync *sync.MockSync
 var tListener *bufconn.Listener
 var tCtx context.Context
-var tGrpc Config
```
Comment (b00f) [inline www/grpc/server_test.go]: Why did you add gRPC here?

**Review by b00f — CHANGES_REQUESTED** [review]

File: www/grpc/server_test.go (hunk 1 above)
Comment (nagarajmanjunath) [reply www/grpc/server_test.go]: i have removed it.
